### ISSUE TITLE
Item Specific Cooldown Override

### ISSUE THREAD

**Issue by CosmicError** [BiscuitDevelopment/SkyblockAddons#226]
**Problem**: Grappling hook has a cooldown override
**Why (I think)**: acts as a normal item with a right-click ability and special (even though it can't really fish)

**In-depth**: In your cooldownmanager.java file you have 2 get functions, 
```
    private static CooldownEntry get(ItemStack item) {
        return (item == null || !item.hasDisplayName()) ? CooldownEntry.NULL_ENTRY : get(item.getDisplayName());
    }

    private static CooldownEntry get(String itemName) {
        return cooldowns.getOrDefault(itemName, CooldownEntry.NULL_ENTRY);
    }
```
which I assume separate the items between normal Minecraft items and special Hypixel items (with abilities). the problem here though is that typically a special ability is activated with a right-click, but in this case, the fishing rod's (grappling hooks) ability is also activated with a right-click so your code treats it as both a vanilla item and a special Hypixel item therefor displaying a cooldown yet allowing an override. 

**Proposition**: make a special list where this happens and check the item/ItemStack  name against this list so it can whitelist specific items (like the grappling hook)

*I'm still very new to java so most of this is just speculation on where the problem originates, but the problem is real.*

**Comment by DidiSkywalker** [comment]
There's no differentiation between normal and Skyblock items in the CooldownManager, `get(ItemStack)` is just for convenience. The CooldownManager really just puts strings on cooldown.
This is simply caused by putting the Grappling Hook on cooldown again, while it is still on cooldown. 
Same happens with the Jungle Axe. I'll PR a fix for this and other small cooldown issues tomorrow probably. Thanks for reporting and putting in the effort of trying to find the cause tho!

**Comment by DidiSkywalker** [comment]
Biscuit actually fixed this already.
In [this](https://github.com/BiscuitDevelopment/SkyblockAddons/commit/a3727c534a29bfc3c92c22191a11c1691c17a452#diff-8dec3606846dfca8487f40967a0e7e1a) commit if you're interested how

**Comment by CosmicError** [comment]
this is still a problem in v1.4.2 though (couldn’t find the release dat of the commit you gave me)

**Comment by DidiSkywalker** [comment]
Well it's gonna stay a problem in 1.4.2. The commit date was November 24th so the fix will be out in 1.5

**Comment by CosmicError** [comment]
ok
